be/src/exec/parquet_scanner.cpp

```diff
@@ -343,13 +343,31 @@ Status ParquetScanner::next_batch() {
     SCOPED_RAW_TIMER(&_counter->read_batch_ns);
     _batch_start_idx = 0;
     if (_curr_file_reader == nullptr) {
+        if (_last_range_size - _last_file_scan_bytes != 0) {
```

Comment:
Under which circumstances, the `_last_range_size` is not equal `_last_file_scan_bytes` when switch reader?

Reply:
The range that parquet actually reads and the division on FE may be different